CHANGELOG.md

```diff
@@ -4,21 +4,28 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
-## Unreleased
+## 2.5.1 (2021-12-01)
```

Comment:
@mkindahl Don't we keep  the Unreleased section with the Features on master and just move everything else under the section for 2.5.1?
i.e. we still need this section in the CHANGELOG.md file? 
```
# Unreleased
Write Preview
 
Attach files by dragging & dropping, selecting or pasting them.
Start a review
Add single comment
Cancel

**Features**
* #3768 Allow ALTER TABLE ADD COLUMN with DEFAULT on compressed hypertable
* #3769 Allow ALTER TABLE DROP COLUMN on compressed hypertable
```

Reply:
That makes sense, so yes, it should probably be kept with the unreleased rows.

Reply:
Anything not making it into the release should stay in the Unreleased section. Only in the actual release branch you can remove the unreleased section but not on master.